5g-control-plane/values.yaml

```diff
@@ -273,14 +273,16 @@ config:
         configuration:
           nrfUri: http://nrf:29510
           webuiUri: webui:9876
+          enableNrfCaching: true
+          nrfCacheEvictionInterval: 900
           plmnList:
             - plmnId:
                 mcc: "208"
                 mnc: "93"
             - plmnId:
                 mcc: "222"
                 mnc: "88"
-          serviceNameList:
+          serviceList:
```

Comment:
did you change this in the backend ?

Reply:
Yes, this was changed in the `UDM` (https://github.com/omec-project/udm/pull/135/files#diff-7d2ff7ee141e8dcfa4279b4e2349e12cb0a3f1c394ca4957f2f4792e56a6b5a4R42) and that is why I have to change it here